addons/l10n_account_edi_ubl_cii_tests/tests/test_xml_cii_fr.py

```diff
@@ -0,0 +1,272 @@
+# -*- coding: utf-8 -*-
+
+from odoo.addons.l10n_account_edi_ubl_cii_tests.tests.common import TestUBLCommon
+from odoo.tests import tagged
+
+@tagged('post_install_l10n', 'post_install', '-at_install')
+class TestCIIFR(TestUBLCommon):
+
+    @classmethod
+    def setUpClass(cls,
+                   chart_template_ref="l10n_fr.l10n_fr_pcg_chart_template",
+                   edi_format_ref="account_edi_facturx.edi_facturx_1_0_05",
+                   ):
+        """
+            this test will fail if account_edi_facturx is not installed. In order not to duplicate the
+            account.edi.format already installed, we use the existing ones (comprising
+            account_edi_facturx.facturx_1_0_05).
+        """
+        super().setUpClass(chart_template_ref=chart_template_ref, edi_format_ref=edi_format_ref)
+
+        cls.partner_1 = cls.env['res.partner'].create({
+            'name': "partner_1",
+            'street': "Rue Jean Jaurès, 42",
+            'zip': "75000",
+            'city': "Paris",
+            'vat': 'FR05677404089',
+            'country_id': cls.env.ref('base.fr').id,
+            'bank_ids': [(0, 0, {'acc_number': 'FR15001559627230'})],
+            'phone': '+1 (650) 555-0111',
+            'email': "partner1@yourcompany.com",
+            'ref': 'seller_ref',
+        })
+
+        cls.partner_2 = cls.env['res.partner'].create({
+            'name': "partner_2",
+            'street': "Rue Charles de Gaulle",
+            'zip': "52330",
+            'city': "Colombey-les-Deux-Églises",
+            'vat': 'FR35562153452',
+            'country_id': cls.env.ref('base.fr').id,
+            'bank_ids': [(0, 0, {'acc_number': 'FR90735788866632'})],
+            'ref': 'buyer_ref',
+        })
+
+        cls.tax_21 = cls.env['account.tax'].create({
+            'name': 'tax_21',
+            'amount_type': 'percent',
+            'amount': 21,
+            'type_tax_use': 'sale',
+            'country_id': cls.env.ref('base.fr').id,
+        })
```

Comment:
why don't you use the taxes generating by the french coa?

Reply:
Yep, I forgot to remove them (when I was using the generic CoA), done now